### ISSUE TITLE
Error: Cannot extract value when objectId is given

### ISSUE THREAD

**Issue by rgaudin** [webrecorder/browsertrix-crawler#329]
I've encountered several occurrences of this error recently and I wonder if it's a known pattern and if there is something to know about this.

Here's the tail of the log ; this occurred after more than an hour

```
Running browsertrix-crawler crawl: crawl --failOnFailedSeed --waitUntil load --depth -1 --timeout 90 --behaviors autoplay,autofetch,siteSpecific --behaviorTimeout 90 --sizeLimit 4294967296 --diskUtilization 90 --timeLimit 7200 --url https://swcw.org.uk/wiki --userAgentSuffix Youzim.it+contact+zimfarm@kiwix.org --cwd /output/.tmpkn75wiq8 --statsFilename /output/crawl.json
```

```
{"logLevel":"info","timestamp":"2023-06-01T12:33:41.107Z","context":"pageStatus","message":"Page Finished","details":{"loadState":4,"page":"https://swcw.org.uk/testwiki/index.php?title=Development&action=history","workerid":0}}
{"logLevel":"info","timestamp":"2023-06-01T12:33:41.134Z","context":"worker","message":"Starting page","details":{"workerid":0,"page":"https://swcw.org.uk/wiki/Special:WhatLinksHere/Development"}}
{"logLevel":"info","timestamp":"2023-06-01T12:33:41.136Z","context":"crawlStatus","message":"Crawl statistics","details":{"crawled":1743,"total":15808,"pending":1,"failed":35,"limit":{"max":0,"hit":false},"pendingPages":["{\"seedId\":0,\"started\":\"2023-06-01T12:33:41.134Z\",\"url\":\"https://swcw.org.uk/wiki/Special:WhatLinksHere/Development\",\"added\":\"2023-06-01T11:27:50.947Z\",\"depth\":2}"]}}
{"logLevel":"info","timestamp":"2023-06-01T12:33:41.296Z","context":"general","message":"Awaiting page load","details":{"page":"https://swcw.org.uk/wiki/Special:WhatLinksHere/Development","workerid":0}}
{"logLevel":"info","timestamp":"2023-06-01T12:33:42.872Z","context":"behavior","message":"Running behaviors","details":{"frames":1,"frameUrls":["https://swcw.org.uk/wiki/Special:WhatLinksHere/Development"],"page":"https://swcw.org.uk/wiki/Special:WhatLinksHere/Development","workerid":0}}
{"logLevel":"info","timestamp":"2023-06-01T12:33:42.872Z","context":"behavior","message":"Run Script Started","details":{"frameUrl":"https://swcw.org.uk/wiki/Special:WhatLinksHere/Development","page":"https://swcw.org.uk/wiki/Special:WhatLinksHere/Development","workerid":0}}
{"logLevel":"info","timestamp":"2023-06-01T12:33:42.881Z","context":"behavior","message":"Run Script Finished","details":{"frameUrl":"https://swcw.org.uk/wiki/Special:WhatLinksHere/Development","page":"https://swcw.org.uk/wiki/Special:WhatLinksHere/Development","workerid":0}}
{"logLevel":"info","timestamp":"2023-06-01T12:33:42.882Z","context":"behavior","message":"Behaviors finished","details":{"finished":1,"page":"https://swcw.org.uk/wiki/Special:WhatLinksHere/Development","workerid":0}}
{"logLevel":"info","timestamp":"2023-06-01T12:33:42.883Z","context":"pageStatus","message":"Page Finished","details":{"loadState":4,"page":"https://swcw.org.uk/wiki/Special:WhatLinksHere/Development","workerid":0}}
{"logLevel":"info","timestamp":"2023-06-01T12:33:42.911Z","context":"worker","message":"Starting page","details":{"workerid":0,"page":"https://swcw.org.uk/testwiki/index.php?title=Development&printable=yes"}}
{"logLevel":"info","timestamp":"2023-06-01T12:33:42.912Z","context":"crawlStatus","message":"Crawl statistics","details":{"crawled":1744,"total":15828,"pending":1,"failed":35,"limit":{"max":0,"hit":false},"pendingPages":["{\"seedId\":0,\"started\":\"2023-06-01T12:33:42.910Z\",\"url\":\"https://swcw.org.uk/testwiki/index.php?title=Development&printable=yes\",\"added\":\"2023-06-01T11:27:50.948Z\",\"depth\":2}"]}}
{"logLevel":"info","timestamp":"2023-06-01T12:33:43.059Z","context":"general","message":"Awaiting page load","details":{"page":"https://swcw.org.uk/testwiki/index.php?title=Development&printable=yes","workerid":0}}
file:///app/node_modules/puppeteer-core/lib/esm/puppeteer/util/assert.js:25
        throw new Error(message);
              ^

Error: Cannot extract value when objectId is given
    at assert (file:///app/node_modules/puppeteer-core/lib/esm/puppeteer/util/assert.js:25:15)
    at valueFromRemoteObject (file:///app/node_modules/puppeteer-core/lib/esm/puppeteer/common/util.js:212:5)
    at createClientError (file:///app/node_modules/puppeteer-core/lib/esm/puppeteer/common/util.js:95:16)
    at CDPPage._CDPPage_handleException (file:///app/node_modules/puppeteer-core/lib/esm/puppeteer/common/Page.js:890:62)
    at file:///app/node_modules/puppeteer-core/lib/esm/puppeteer/common/Page.js:183:100
    at file:///app/node_modules/puppeteer-core/lib/esm/third_party/mitt/index.js:1:248
    at Array.map (<anonymous>)
    at Object.emit (file:///app/node_modules/puppeteer-core/lib/esm/third_party/mitt/index.js:1:232)
    at CDPSessionImpl.emit (file:///app/node_modules/puppeteer-core/lib/esm/puppeteer/common/EventEmitter.js:81:22)
    at CDPSessionImpl._onMessage (file:///app/node_modules/puppeteer-core/lib/esm/puppeteer/common/Connection.js:435:18)

Node.js v18.16.0

```

[full log](https://s3.us-west-1.wasabisys.com/org-kiwix-zimit/d1b9b79e-4629-4f87-90a3-e21a789948d0_zimit.log)

**Comment by ikreymer** [comment]
Are you using 0.10.0 or 0.10.1? Hopefully this was fixed in 0.10.1 by updating puppeteer, see #324 and https://github.com/puppeteer/puppeteer/issues/10224

**Comment by rgaudin** [comment]
Was `0.10.0`. Closing ✅
